xivo_dao/resources/utils/search.py

```diff
@@ -13,10 +13,15 @@
 from sqlalchemy import sql
 from sqlalchemy.sql.functions import ReturnTypeFromArgs
 from sqlalchemy.types import Integer
+import re
 
 from xivo_dao.helpers import errors
 
 
+def contains_latin(text):
+    return bool(re.search(r'[a-zA-Z]', text))  # Check if the text has Latin characters
```

Comment:
Not sure I understand this implementation.
Wouldn't the regex match non-accented characters only?
If there are latin accented characters, and we want to `unidecode` those, we want to make sure this `contains_latin` match those strings.

we would need additional test coverage to ensure the behavior with latin characters (accented or not) is retained